templates/kommander.yaml

```diff
@@ -10,12 +10,15 @@ metadata:
     # on the cluster, this hack will trigger re-queue on Addons until one exists.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    appversion.kubeaddons.mesosphere.io/kommander: "1.50.0"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.114.5"
     endpoint.kubeaddons.mesosphere.io/kommander: /ops/portal/kommander
-    values.chart.helm.kubeaddons.mesosphere.io/kommander: "https://raw.githubusercontent.com/mesosphere/charts/6c43b8ab10108fb1adba5c6dd10e800e5f1abdd0/stable/kommander/values.yaml"
+    values.chart.helm.kubeaddons.mesosphere.io/kommander: "https://raw.githubusercontent.com/mesosphere/charts/dev/stable/kommander/values.yaml"
```

Comment:
@alejandroEsc I think this should point to the master commit instead to `dev` branch

Reply:
Thanks @alejandroEsc